sdk/core/Azure.Core/src/Messaging/MessageContent.cs

```diff
@@ -5,12 +5,12 @@
 using System.ComponentModel;
 using Azure.Core;
 
-namespace Azure
+namespace Azure.Messaging
 {
     /// <summary>
-    /// Content containing a content type along with its data.
+    /// Message containing a content type along with its data.
```

Comment:
```suggestion
    /// The content of a message, comprised of the message data and an identifier for the content type.
```

Seems weird to have a class called `MessageContent` and then describe it as a "Message";  feel free to tune the suggestion text as you see fit or ignore it.